set.mm

```diff
@@ -460080,6 +460128,40 @@ Class abstractions (a.k.a. class builders)
       EBCRUIUOUEUBUFMUG $.
   $}
 
+  ${
+    $d A x $.  $d P x $.  $d X x $.
+    $( The other element of a pair is an element of the pair.  (Contributed by
+       Thierry Arnoux, 26-Aug-2017.) $)
+    unidifsnel $p |- ( ( P C_ A /\ X e. P /\ P ~~ 2o )
+                  -> U. ( P \ { X } ) e. A ) $=
```

Comment:
Interesting: I was not aware of the idiom ` U. ( P \ { X } ) ` for "the other element" of a pair ` P ` until now. Unfortunately, this is nowhere explicitly explained in set.mm, only implicitly by ~en2eleq.

Reply:
This is thanks to Stefan O'Rear I believe. Very clever!
I've made a new version which I think is slightly simpler, so possibly more reusable.